### ISSUE TITLE
During 'verdi code setup/duplicate', any # symbol in the prepend-text crashes the procedure

### ISSUE THREAD

**Issue by borellim** [aiidateam/aiida_core#1965]
Say, for instance, that you use this as prepend-text during 'verdi code setup' or 'verdi code duplicate':
```bash
module load daint-gpu
#module load not-needed
```

or maybe this:
```bash
module load daint-gpu  # a comment
```

Then, after exiting the editor, you will get this:
```
Traceback (most recent call last):
  File "/home/mborelli/venvs/aiida_dev/bin/verdi", line 9, in <module>
    sys.exit(verdi())
  File "/home/mborelli/venvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/mborelli/venvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/mborelli/venvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/mborelli/venvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/mborelli/venvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/mborelli/venvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/mborelli/repos/aiida_dev/aiida-core/aiida/cmdline/utils/decorators.py", line 62, in decorated_function
    return function(*args, **kwargs)
  File "/home/mborelli/repos/aiida_dev/aiida-core/aiida/cmdline/commands/cmd_code.py", line 101, in setup_code
    pre, post = ensure_scripts(kwargs.pop('prepend_text', ''), kwargs.pop('append_text', ''), kwargs)
  File "/home/mborelli/repos/aiida_dev/aiida-core/aiida/cmdline/utils/multi_line_input.py", line 20, in ensure_scripts
    return edit_pre_post(pre, post, summary)
  File "/home/mborelli/repos/aiida_dev/aiida-core/aiida/cmdline/utils/multi_line_input.py", line 39, in edit_pre_post
    pre, post = [text.strip() for text in stripped_input.split('#')]
ValueError: too many values to unpack
```

**Comment by yakutovicha** [comment]
Thanks @borellim  for reporting the issue. The same error is raised when setting up a computer

```
verdi computer setup
Info: enter "?" for help
Computer label: test_computer
Hostname: localhost
Description []: no description
Enable the computer? [True]: 
Transport plugin: local
Scheduler plugin: direct
Shebang line (first line of each script, starting with #!) [#!/bin/bash]: 
Work directory on the computer [/scratch/{username}/aiida/]: 
Mpirun command [mpirun -np {tot_num_mpiprocs}]:  
Default number of CPUs per machine: 2
Traceback (most recent call last):
  File "/home/epfl/work/aiida_core/test-aiida/bin/verdi", line 9, in <module>
    sys.exit(verdi())
  File "/home/epfl/work/aiida_core/test-aiida/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/epfl/work/aiida_core/test-aiida/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/epfl/work/aiida_core/test-aiida/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/epfl/work/aiida_core/test-aiida/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/epfl/work/aiida_core/test-aiida/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/epfl/work/aiida_core/test-aiida/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/epfl/work/aiida_core/test-aiida/local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/epfl/work/aiida_core/aiida/cmdline/utils/decorators.py", line 70, in decorated_function
    return function(*args, **kwargs)
  File "/home/epfl/work/aiida_core/aiida/cmdline/commands/cmd_computer.py", line 194, in computer_setup
    pre, post = ensure_scripts(kwargs.pop('prepend_text', ''), kwargs.pop('append_text', ''), kwargs)
  File "/home/epfl/work/aiida_core/aiida/cmdline/utils/multi_line_input.py", line 20, in ensure_scripts
    return edit_pre_post(pre, post, summary)
  File "/home/epfl/work/aiida_core/aiida/cmdline/utils/multi_line_input.py", line 39, in edit_pre_post
    pre, post = [text.strip() for text in stripped_input.split('#')]
ValueError: too many values to unpack

```

**Comment by sphuber** [comment]
Fixed in PR #1986
